upgrading/automated_upgrades.adoc

```diff
@@ -73,6 +73,8 @@ nodes and thousands of projects and pods, review
 xref:special-considerations-for-large-scale-upgrades[Special considerations for
 large-scale upgrades] to prevent upgrade failures.
 
+* If you are completing a disconnected cluster update, you must update your image registry with new image versions or the cluster update will fail. For example, when updating from `3.11.153` to `3.11.157`, make sure the `3.11.157` image tags are present.
+
```

Comment:
missing 'v' prefix, should be 'v3.11.57'

Reply:
@jianlinliu Thanks for the catch; I didn't realize the `v` was required. I've updated. Please verify this looks good now.

Reply:
LGTM.